### DISCUSSION TITLE
Remove number of drops penalty

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18193] [Feature requests]
I believe we should either remove number of drops stat penalty or remove it on certain ride types or only apply it to drops over a certain height ie 50 ft or more

**Comment by Lastorder-DC** [comment]
...and Why? Please give valid reason to change stuff.

also what is `certain ride types`? This idea is too vauge.

**Comment by 733737** [comment]
Why? The reason why is because I believe it was an ill conceived method for deriving intensity in the first place. There are lots of real life examples of coasters with various drops that do not add intensity

**Comment by ocalhoun6** [comment]
Perhaps instead of using the number of drops, which can unfairly penalize a series of small drops, it should base the calculation on 'total height of all drops'.

**Reply by 733737** [reply]
I think that could be a good middle ground or compromise, as long as we could get the value close to vanilla, it should improve the game!
